### PR TITLE
Milestone 6 live page

### PR DESCRIPTION
**Pull request by eroiyo**
Add a Hyperlink in the README.MD to go to the live version of GitHub

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Download it from this repository, and double click into index.html
 
 👤 **Author1**
 
-- GitHub: [@eroiyo](https://github.com/eroiyo)
+- GitHub: [@eroiyo](https://eroiyo.github.io/My-Portafolio/)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/carlos-arturo-ortega-guanipa-39a1a5204/)
 
 ## 🤝 Contributing
```
Comment (Temmarie) [inline README.md:28]: - [ ] A good README should have a description that is clear, short, and to the point. It should describe the importance of your project, what it does and how to use it. This way technical and non-technical users can understand how to use it.  Refer to this template [here](https://github.com/microverseinc/readme-template) and customise it to your needs. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)). 
- Focus on the instructions you gave for cloning the repo and running it locally.
- The live link in your project leads to the GitHub error page.
 image

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @eroiyo and @Ismailco    ! :wave: :wave:

#### You have done really well in implementing the requirements.
- You used proper GitHub Flow ✔️ 
- Your linter checks are all passing ✔️ 

 However, there are changes required for you to make in order for this project to be approved. 

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. :smile:
- - -

### **Required Changes** ♻️
- [ ] Your live link is broken, kindly fix it.
_See the inline comments below for more required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- N/A

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**Note that you still have 2 more reviews as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

__Kindly make the necessary changes and submit another code review request.__

**Comment by eroiyo** [comment]
Thanks for the feedback!

Fixed the broken link, and instructions added

**Review by DeliceLydia — APPROVED** [review]
# Congratulations 🎉🎉

Hi @eroiyo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**